test/jdk/sun/net/www/http/HttpClient/MultiThreadTest.java

```diff
@@ -139,61 +177,98 @@ public static void main(String args[]) throws Exception {
         Server svr = new Server(ss);
         svr.start();
 
-        Object lock = MultiThreadTest.getLock();
-        synchronized (lock) {
-            for (int i=0; i<threads; i++) {
-                MultiThreadTest t = new MultiThreadTest(svr.getAuthority(), requests);
-                t.start ();
+        MAIN_START = System.nanoTime();
+        try {
+            Object lock = MultiThreadTest.getLock();
+            List<MultiThreadTest> tests = new ArrayList<>();
```

Comment:
this list is not used

Reply:
Good catch. That's a left over from some additional experiment.

Reply:
I decided to follow up on your (offline) suggestion and actually join the client threads (later). I have also added a countdown latch to make the wait cleaner. As a result the END atomic long is no longer needed and a simple local variable can be used instead.